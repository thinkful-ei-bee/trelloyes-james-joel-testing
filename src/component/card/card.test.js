import React from 'react';
import ReactDOM from 'react-dom';
import STORE from '../../store/store';

import Card from './card';

describe('<Card />', () => {
    it('renders without crashing', () => {
        // Create a DOM element to render the component into
        const div = document.createElement('div');

        // Render the component
        // If something is wrong it will fail here
        ReactDOM.render(<Card allCards={STORE.allCards} cardIds={STORE.lists[0].cardIds}/>, div);

        // Clean up
        ReactDOM.unmountComponentAtNode(div);
    });
});