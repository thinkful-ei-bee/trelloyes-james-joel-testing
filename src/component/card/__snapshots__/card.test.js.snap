// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders without crashing 1`] = `
<div
  className="List-cards"
>
  <div
    className="Card"
  >
    <h3>
      First card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Second card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Fifth card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Sixth card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Seventh card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Tenth card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Twelth card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
  <div
    className="Card"
  >
    <h3>
      Thirteenth card
    </h3>
    <p>
      lorem ipsum
    </p>
  </div>
</div>
`;
